lexicons/app/bsky/unspecced/getPostThreadHiddenV2.json

```diff
@@ -0,0 +1,74 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.unspecced.getPostThreadHiddenV2",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "(NOTE: this endpoint is under development and WILL change without notice. Don't use it until it is moved out of `unspecced` or your application WILL break) Get the hidden posts in a thread. It is based in an anchor post at any depth of the tree, and returns hidden replies (recursive replies, with branching to their replies) below the anchor. It does not include ancestors nor the anchor. This should be called after exhausting `app.bsky.unspecced.getPostThreadV2`. Does not require auth, but additional metadata and filtering will be applied for authed requests.",
+      "parameters": {
+        "type": "params",
+        "required": ["anchor"],
+        "properties": {
+          "anchor": {
+            "type": "string",
+            "format": "at-uri",
+            "description": "Reference (AT-URI) to post record. This is the anchor post."
+          }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["thread"],
+          "properties": {
+            "thread": {
+              "type": "array",
+              "description": "A flat list of thread hidden items. The depth of each item is indicated by the depth property inside the item.",
+              "items": {
+                "type": "ref",
+                "ref": "#threadHiddenItem"
+              }
+            }
+          }
+        }
+      }
+    },
+    "newest": { "type": "token" },
+    "oldest": { "type": "token" },
+    "top": { "type": "token" },
+    "threadHiddenItem": {
+      "type": "object",
+      "required": ["uri", "depth", "value"],
+      "properties": {
+        "uri": {
+          "type": "string",
+          "format": "at-uri"
+        },
+        "depth": {
+          "type": "integer",
+          "description": "The nesting level of this item in the thread. Depth 0 means the anchor item. Items above have negative depths, items below have positive depths."
+        },
+        "value": {
+          "type": "ref",
+          "ref": "#threadHiddenItemPost"
+        }
```

Comment:
I notice that the analogous property from `getPostThreadV2` was a union.  How likely is it that this will become a union?  If it's likely, we might get ahead of it now.  I suppose this is unspecced so we have some wiggle-room, low-stakes.